CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Changes
```

Comment:
When versioning it's easier to see if we have only "bug fixes" or also "features" to chose to bump a minor or patch:

```suggestion
### Fixes
```